src/helm/benchmark/scenarios/opinions_qa_scenario.py

```diff
@@ -0,0 +1,170 @@
+    CODALAB_URI_TEMPLATE: str = (
+        "https://worksheets.codalab.org/rest/bundles/{bundle}/contents/blob/model_input/{filename}"
+    )
+    CODALAB_BUNDLE: str = "0xa6f81cc62d7d4ccb93031a72d2043669"
+    FILE_NAME: str = "Pew_American_Trends_Panel_W{wave}.csv"
+    PEW_SURVEY_WAVES: list = [26, 27, 29, 32, 34, 36, 41, 42, 43, 45, 49, 50, 54, 82, 92]
+
+    def __init__(self, survey_type: str, context: str):
+        super().__init__()
+        assert context in ["default", "steer-qa", "steer-bio", "steer-portray"]
+
+        self.survey_type: str = survey_type
+        self.context: str = context
+
+    def download_data(self):
+
+        DOWNLOAD_FILENAMES = [self.FILE_NAME.format(wave=wave) for wave in self.PEW_SURVEY_WAVES]
+        DOWNLOAD_FILENAMES += [f"{steer}.csv" for steer in ["steer-qa", "steer-bio", "steer-portray"]]
+        DOWNLOAD_FILENAMES += ["Pew_American_Trends_Panel_disagreement_500.csv"]
+
+        for filename in DOWNLOAD_FILENAMES:
+            data_path: str = os.path.join(self.output_path, filename)
+
+            source_url: str = self.CODALAB_URI_TEMPLATE.format(bundle=self.CODALAB_BUNDLE, filename=filename)
+            if not os.path.exists(data_path):
+                shell(["wget", source_url, "--no-check-certificate", "-O", data_path])
+
+    def read_survey_questions(self, csv_path):
+        df = pd.read_csv(csv_path, sep="\t")
+        df["options"] = df.apply(lambda x: eval(x["options"]), axis=1)
+        return df
+
+    def get_instances(self) -> List[Instance]:
+
+        self.output_path: str = os.path.join(self.output_path, "data")
+        if not os.path.exists(self.output_path):
+            os.makedirs(self.output_path)
+
+        self.download_data()
+
+        # Read all the instances
+        instances: List[Instance] = []
+        splits: Dict[str, str] = {
+            "dev": TRAIN_SPLIT,
+            "test": TEST_SPLIT,
+        }
+
+        all_splits = ["dev", "test"] if self.context == "steer-qa" else ["test"]
+        csv_dict = {
+            "dev": os.path.join(self.output_path, f"{self.context}.csv"),
+            "test": os.path.join(self.output_path, f"{self.survey_type}.csv"),
+        }
+
+        bios_df = None
+        if self.context in ["steer-bio", "steer-portray"]:
+            bios_path = os.path.join(self.output_path, f"{self.context}.csv")
+            bios_df = pd.read_csv(bios_path, sep="\t")
+
+        for split in all_splits:
+
+            csv_path: str = csv_dict[split]
+            assert os.path.exists(csv_path)
+
+            question_df = self.read_survey_questions(csv_path)
+
+            for qidx, (question, answers) in enumerate(zip(question_df["question"], question_df["options"])):
+
+                # Opinions QA test questions have no correct answer. However, since the HELM codebase requires a
+                # correct answer to be associated with each instance, we set it to be the first reference.
```

Comment:
Does it?  I think you should be able to have no CORRECT_TAG on any of the references.  And then if something is crashing, we should fix it there, not here.

Reply:
I think it tries to compute basic metrics by default, and these metrics require a correct answer. This was true at least in the older version of helm.

Reply:
Actually, I think I should just not be calculating any metrics so I should be able to remove this.